src/auth/auth.module.ts

```diff
@@ -0,0 +1,31 @@
+import { Module } from '@nestjs/common';
+import { AuthController } from './controller/auth.controller';
+import { AuthService } from './service/auth.service';
+import { CustomersModule } from '../customers/customers.module';
+import { JwtModule } from '@nestjs/jwt';
+import { JwtStrategy } from './strategies/jwt.strategy';
+import { RefreshJwtStrategy } from './strategies/refresh-token.strategy';
+import { RolesGuard } from './guard/roles.guard';
+import { LocalStrategy } from './strategies/local.strategy';
+
+@Module({
+  controllers: [AuthController],
+  providers: [
+    AuthService,
+    JwtStrategy,
+    RefreshJwtStrategy,
+    RolesGuard,
+    LocalStrategy,
+  ],
+  imports: [
+    CustomersModule,
+    JwtModule.register({
+      global: true,
+      secret: '$2a$12$woZJtsABa4SaU4KgOKCfaO5NHM6/jugg.A.HaJnCGCbn5i3ZrHECG',
```

Comment:
For this training, it is ok to add the secret like this.

However, a better way would be to read it from `.env`  via `ConfigModule` and not push that file to GitHub (via `.gitignore`). In real projects, we should neverexpose secrets by making them available on the internet. 

The same thing applies to the DB credentials from `app.module.ts`.

You can do these changes if you have some time today. Do not forget to modify `src/auth/strategies/jwt.strategy.ts` too.

References: 
- https://docs.nestjs.com/techniques/configuration
- https://docs.nestjs.com/techniques/database#async-configuration